src/libs/ajax/file-browser-providers/GCSFileBrowserProvider.ts

```diff
@@ -34,7 +34,7 @@ const GCSFileBrowserProvider = ({ bucket, project, pageSize = 1000 }: GCSFileBro
 
     if (nextPageToken || isFirstPage) {
       do {
-        const requestOptions: { maxResults: number, pageToken?: string } = {
+        const requestOptions: { maxResults: number; pageToken?: string } = {
```

Comment:
declaring this type beforehand would be better I think.

Reply:
I think this comment got lost in the PR shuffle.  This file looks to not have the comment addressed.  Inline types like this are harder to read.  It would be better to define RequestOptions type or interface just before this usage.

Reply:
This file is no longer part of this PR. It was part of the ESLint configuration change that was split out into #3496.